### ISSUE TITLE
Visualizing the Dataset

### ISSUE THREAD

**Issue by Naxalov** [Naxalov/PyTorch_VGG#4]
Visualizing the Dataset
- [x] Visualizations single dataset (Display image and label.)
- [x] Creating a Grid Visualization
- [x] Visualizations single dataloader (Display image and label.)
